tests/end-to-end.py

```diff
@@ -1147,6 +1151,89 @@ def __init__(
     self._image_export_path = None
     self._InitializeImageExportPath()
 
+  def _CompareHashesFile(self, test_definition, temp_directory):
+    """Compares the hashes file with a reference hashes file.
+
+    Args:
+      test_definition (TestDefinition): test definition.
+      temp_directory (str): name of a temporary directory.
+
+    Returns:
+      bool: True if he hashes files are identical.
+    """
+    hashes_file_path = os.path.join(temp_directory, test_definition.hashes_file)
+
+    result = False
+    if test_definition.reference_hashes_file:
+      reference_hashes_file_path = test_definition.reference_hashes_file
+      if self._test_references_path:
+        reference_hashes_file_path = os.path.join(
+            self._test_references_path, reference_hashes_file_path)
+
+      if not os.path.exists(reference_hashes_file_path):
+        logging.error('No such reference hashes file: {0:s}'.format(
+            reference_hashes_file_path))
+        return False
+
+      with open(reference_hashes_file_path, 'r') as reference_hashes_file:
+        with open(hashes_file_path, 'r') as hashes_file:
+          # Hack to remove paths in the hashes that are different when running
```

Comment:
It looks like on windows the path & hashes are different than what is expected.
SHould we have different reference hash files per platform?

Reply:
hashes should be the same, paths are different. I currently reformat the paths.

Reply:
I see what you mean, I've changed the code a bit.